src/main/java/org/example/student/Student.java

```diff
@@ -0,0 +1,102 @@
+package org.example.student;
+
+import org.example.course.Course;
+import org.example.department.Department;
+
+import java.util.Set;
+
+public class Student {
+
+    public enum Gender {
+        MALE,
+        FEMALE
+    }
+
+    private String name;
+    private String surname;
+    private String studentId;
+    private Department department;
+    private Set<Course> courses;
+    private int age;
+    private Gender gender;
+
+    public Student() {
```

Comment:
This is useless -  I guess you always want to create a student with name, surname, id age and Gender